src/main/kotlin/no/skatteetaten/aurora/boober/service/DeployLogService.kt

```diff
@@ -43,8 +43,17 @@ class DeployLogService(
                         result = DeployHistoryEntryResult(result.openShiftResponses, result.tagResponse),
                         projectExist = result.projectExist
                     )
-                    val storeResult = storeDeployHistory(deployHistory, result.auroraDeploymentSpecInternal!!.cluster)
-                    it.copy(bitbucketStoreResult = storeResult)
+                    try {
+                        val storeResult =
+                            storeDeployHistory(deployHistory, result.auroraDeploymentSpecInternal!!.cluster)
+                        it.copy(bitbucketStoreResult = storeResult)
+                    } catch (e: Exception) {
+                        it.copy(
+                            bitbucketStoreResult = e.localizedMessage,
```

Comment:
I may be missing something, but is bitbucketStoreResult part of the final response?

Reply:
yes